tests/system_tests/test_artifacts/test_wandb_artifacts_api.py

```diff
@@ -455,3 +456,45 @@ def test_run_log_artifact(wandb_init):
     actual_artifacts = list(run.logged_artifacts())
     assert len(actual_artifacts) == 1
     assert actual_artifacts[0].qualified_name == artifact.qualified_name
+
+
+def test_artifact_enable_tracking_flag(user, wandb_init):
+    """Test that enable_tracking flag is correctly passed through the API chain."""
+    api = Api()
```

Comment:
**suggestion (non-blocking):** 
- use the existing `api` fixture (should be ok since this is a system test)
- use the `mocker` fixture instead of `patch`-ing ([docs](https://pytest-mock.readthedocs.io/en/stable/usage.html)) -- see other suggestion below
```suggestion
def test_artifact_enable_tracking_flag(user, wandb_init, api, mocker):
    """Test that enable_tracking flag is correctly passed through the API chain."""
```